docs/Getting-Started/Create-bootable-media/create-bootable-usb.md

```diff
@@ -0,0 +1,34 @@
+# Create Pop!_OS Installation Media
+
+Pop!_OS can be run from a USB drive in a live environment. You can use the live USB to install Pop!_OS permenantly on a system, demo Pop!_OS without installing, or use the drive as a system recovery tool.
+
+---
+## Preparation
+
+Creating a bootable Pop!_OS USB drive requires three components: 
+
+- A USB flash drive. The flash drive must have at least 8 GB of free space. All data on the flash drive will be lost during the creation process.
+
+- The Pop!_OS.iso image: [Download Pop!_OS at this link.](https://pop.system76.com/)
+
+- A program to write the Pop!_OS.iso image to the flash drive. [Etcher](https://www.balena.io/etcher/) and [Popsicle](https://github.com/pop-os/popsicle) are popular options for Linux users, and [Rufus](https://rufus.ie/en/) is compatible with both Windows and macOS.
```

Comment:
Etcher is cross-platform (Windows, Mac and Linux) which is why we usually recommend it for non Pop users.
Popsicle being an AppImage also means that it is available for all (most) Linux distros.